### DISCUSSION TITLE
CoPilot.nvim seems to trip over comment auto insertion.

### DISCUSSION THREAD

**Discussion by thedracle** [community/community#7073] [Copilot Conversations] [answered]
One thing I have noticed is the suggestions seem to get interrupted, or race, with the fact Vim automatically inserts/continues comments.

I spent some time backspacing the inserted comments and would get the rest of the suggestions quite nicely.

Setting:
```
autocmd FileType * setlocal formatoptions-=c formatoptions-=r formatoptions-=o
```

In my init.vim helps a lot.

Pretty much anything that inserts characters into the buffer seems to cause issues with partially complete suggestions, so I've disabled them.

Being able to cycle through suggestions would really be helpful to making CoPilot capable of replacing the intended behavior (maybe cycle through to a suggestion that continues the comment for instance?)

**Comment by tpope** [comment, marked as answer]
You can probably leave `c` enabled without downside but the others might indeed help.

I wouldn't expect cycling to help. There aren't any suggestions to cycle through.
